var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -19,6 +19,8 @@ class Mumps(Package):
     variant('float', default=True, description='Activate the compilation of smumps')
     variant('complex', default=True, description='Activate the compilation of cmumps and/or zmumps')
     variant('idx64', default=False, description='Use int64_t/integer*8 as default index type')
+    variant('tests', default=False, description='Run small MPI unit tests')
```

Comment:
@adamjstewart i see. Let's do it step by step: I removed test variant and for a moment I run test only in non-MPI mode. After it's clear what/how tests should work, it should be straightforward to enable tests in MPI case as well.